__tests__/config/fixtures.ts

```diff
@@ -108,11 +109,28 @@ export const createBlockForDevnet = async (): Promise<void> => {
   await fetch(new URL('/create_block', process.env.TEST_RPC_URL), { method: 'POST' });
 };
 
+export async function waitNextBlock(provider: RpcProvider, delay: number) {
```

Comment:
Maybe we should add this to Provider?
Default wait for +1 block if no identifier provided ?
`waitForBlock(blockIdentifier?: BlockIdentifier)`

Reply:
OK. I start to implement this proposal.

Reply:
Nevertheless, I ask me if it will work properly in Devnet-rs or Katana....

Reply:
Implemented.
Tested in Sepolia.
To see if we keep the last test (can have a very long duration).

Reply:
can this be removed now and `provider.waitForBlock` be used in tests?

Reply:
Unfortunately not.
As the Devnet-rs used in CI (v0.0.4) is not creating a block at a periodic duration, this function `waitNextBlock()` includes a function to creates a new bloc manually if we are in devnet, with `createBlockForDevnet()`.

When we will use in Starknet.js CI a Devnet-rs that is configured to create a block each 2-3 sec, it will be possible to remove it.

Reply:
On this subject, do not forget my above comment : Devnet-rs used in CI has to be bumped to v0.0.5 to be able to forget completely Goerli. It seems that it can be made only by a SpaceShard guy.

Reply:
ohh sorry, will do it ASAP